resources/xsd/fastRTPS_profiles.xsd

```diff
@@ -1064,7 +1064,7 @@
         ├ disable_heartbeat_piggyback [0~1],
```

Comment:
Remove `NOT SUPPORTED YET: COMMENTED` comment from previous `destination_order` element.